Doc/extending/embedding.rst

```diff
@@ -275,6 +275,91 @@ will depend on the details of the C++ system used; in general you will need to
 write the main program in C++, and use the C++ compiler to compile and link your
 program.  There is no need to recompile Python itself using C++.
 
+.. _freezingandembeddingmanually:
+
+Freezing Modules in Manually Embedded Python
+============================================
+
+While it is possible to use frozen modules in embedded python if the main module
+is also frozen. It seems people are bitten when they don't have their main module
+frozen and use ``PyRun_SimpleString`` that contains their main module(s) but
+also depend on an frozen module (like the ``__hello__`` module). However if we
+wanted to add our own module to the frozen list we would normally do this::
+
+    #include <importlib.h>
```

Comment:
I believe that [per devguide](https://devguide.python.org/documentation/style-guide/#use-of-whitespace) this should be three spaces, as the indentation is a reST indentation not C.

Reply:
![image](https://user-images.githubusercontent.com/15173749/197342097-0eaf66d8-f475-42df-a370-bda755070cfc.png)

Reply:
However, the spaces could be changed though to do the 2 spaces per sentence.

Reply:
I mean in the sense of the initial spacing being 3, then 4 spaces for inside the code block itself (i.e., see around L56 `#define PY_SSIZE_T_CLEAN`).